facebook/OpenBIC
# fby35: cl: Support ipmi get sdr OEM command

Summary:
- Support ipmi get sdr OEM command to extend length of sensor name.

Test Plan:
- Build code: Pass
- Check sensor name via sensor-util: Pass

Log:
uart:~$ platform sensor list_all
---------------------------------------------------------------------------------
[0x1 ] MB_INLET_TEMP_C          : tmp75      | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 31.0
[0x2 ] MB_OUTLET_TEMP_C         : tmp431     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 58.437
[0x3 ] FIO_FRONT_TEMP_C         : tmp75      | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 26.0
[0xd ] MB_SSD0_M2A_TEMP_C       : nvme       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 27.0
[0x5 ] MB_SOC_CPU_TEMP_C        : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 58.0
[0x14] MB_SOC_THERMAL_MARGIN_C  : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 42.0
[0x15] MB_SOC_TJMAX_C           : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 100.0
[0x6 ] MB_DIMMA0_TEMP_C         : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 56.0
[0x7 ] MB_DIMMA2_TEMP_C         : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 55.0
[0x9 ] MB_DIMMA3_TEMP_C         : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 52.0
[0xa ] MB_DIMMA4_TEMP_C         : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 54.0
[0xb ] MB_DIMMA6_TEMP_C         : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 50.0
[0xc ] MB_DIMMA7_TEMP_C         : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 45.0
[0x38] MB_SOC_PACKAGE_PWR_W     : peci       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 77.0
[0x20] MB_ADC_P12V_STBY_VOLT_V  : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 12.733
[0x22] MB_ADC_P3V3_STBY_VOLT_V  : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 3.348
[0x24] MB_ADC_P1V05_PCH_VOLT_V  : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.59
[0x21] MB_ADC_P3V_BAT_VOLT_V    : adc        | access[O] | poll[O] 3600 sec | 4byte_acur_read_success   | 3.132
[0x25] MB_ADC_P5V_STBY_VOLT_V   : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 4.998
[0x26] MB_ADC_P12V_DIMM_VOLT_V  : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 12.586
[0x27] MB_ADC_P1V2_STBY_VOLT_V  : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.196
[0x28] MB_ADC_P3V3_M2_VOLT_V    : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 3.314
[0x23] MB_ADC_P1V8_STBY_VOLT_V  : adc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.801
[0x2e] MB_VR_VCCD_VOLT_V        : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.142
[0x2f] MB_VR_FAON_VOLT_V        : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.48
[0x2d] MB_VR_EHV_VOLT_V         : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.800
[0x2a] MB_VR_VCCIN_VOLT_V       : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.772
[0x2c] MB_VR_FIVRA_VOLT_V       : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.818
[0x34] MB_VR_VCCD_CURR_A        : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 2.800
[0x35] MB_VR_FAON_CURR_A        : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 9.500
[0x33] MB_VR_EHV_CURR_A         : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 0.600
[0x31] MB_VR_VCCIN_CURR_A       : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 32.300
[0x32] MB_VR_FIVRA_CURR_A       : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 4.0
[0x12] MB_VR_VCCD_TEMP_C        : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 50.0
[0x13] MB_VR_FAON_TEMP_C        : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 54.0
[0x11] MB_VR_EHV_TEMP_C         : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 49.0
[0xf ] MB_VR_VCCIN_TEMP_C       : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 69.0
[0x10] MB_VR_FIVRA_TEMP_C       : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 64.0
[0x3e] MB_VR_VCCD_PWR_W         : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 2.0
[0x3f] MB_VR_FAON_PWR_W         : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 9.0
[0x3d] MB_VR_EHV_PWR_W          : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 0.0
[0x3a] MB_VR_VCCIN_PWR_W        : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 57.0
[0x3c] MB_VR_FIVRA_PWR_W        : isl69259   | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 7.0
[0x4 ] MB_PCH_TEMP_C            : pch        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 55.0
[0x1e] MB_VR_DIMMA0_PMIC_PWR_W  : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.750
[0x1f] MB_VR_DIMMA2_PMIC_PWR_W  : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.375
[0x36] MB_VR_DIMMA3_PMIC_PWR_W  : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.750
[0x37] MB_VR_DIMMA4_PMIC_PWR_W  : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.625
[0x42] MB_VR_DIMMA6_PMIC_PWR_W  : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.500
[0x47] MB_VR_DIMMA7_PMIC_PWR_W  : pmic       | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 1.375
[0xe ] MB_HSC_TEMP_C            : tmp431     | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 54.437
[0x29] MB_HSC_INPUT_VOLT_V      : hsc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 12.495
[0x30] HSC_OUTPUT_CURR_A        : hsc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 7.963
[0x39] MB_HSC_INPUT_PWR_W       : hsc        | access[O] | poll[O] 1    sec | 4byte_acur_read_success   | 98.335
---------------------------------------------------------------------------------
